CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change history for stripes-core
 
+## [CURRENT]
+
+* Added ability to set the current user's service points in the session object. Enables UIU-551.
```

Comment:
Add "Available from v2.11.1", so client code that needs this functionality knows which version to depend on.